src/app/repo-preview/repo-preview.component.html

```diff
@@ -0,0 +1,9 @@
+<div *ngIf="repo">
+  <a href="/">
+    <div class="card">
+      <h3 class="repo-name">{{repo.name}}</h3>
+      <p class="repo-description">Description: {{repo.description}}</p>
```

Comment:
What is the expected behavior when there is no description in the repository?

Reply:
Ah, I added a mapper to handle this in the next ticket but for now I'll update this to print "not available" if there is no description.